src/Tools/DynamoShapeManager/Utilities.cs

```diff
@@ -214,6 +233,32 @@ public static void PreloadAsmFromPath(string preloaderLocation, string asmLocati
             Debug.WriteLine("Successfully loaded ASM binaries");
         }
 
+        /// <summary>
+        /// Attempts to remap a an old LibG path to a new one using a version map.
+        /// We assume that the leaf directory is of the form LibG_[Version].
+        /// </summary>
+        /// <param name="preloaderLocation"></param>
+        /// <returns> returns "" if the path does not appear to be a valid old libG path.</returns>
```

Comment:
This will be the description of returns, maybe change to `corresponding new version LibG path or empty string`

Reply:
good catch, not sure how that happened.